### DISCUSSION TITLE
Certificate

### DISCUSSION THREAD

**Discussion by Spartan-119** [OpenMined/courses#286] [Course: Our Privacy Opportunity]
I just finished my first course. It was amazing and I had a lot of fun. But I was wondering when can I get the certificate of completion of this course?
Thanks in advance :)

Cheers!

**Comment by znreza** [comment]
@Spartan-119 we are glad to know that you liked it! Have you started the second course of Federated Learning yet? I think you will like it as well.

The team is working hard to prepare certificates for you all. This feature will be added soon! You will be notified when it's ready.

**Reply by Spartan-119** [reply]
@znreza Yeah already aboard the train of federated learning and must confess it is better than I thought. 
It is a comfort to know that the certificates will be distributed in the coming days.
Cheers to the team.

**Reply by NilsMosser** [reply]
> 
> 
> @Spartan-119 we are glad to know that you liked it! Have you started the second course of Federated Learning yet? I think you will like it as well.
> 
> The team is working hard to prepare certificates for you all. This feature will be added soon! You will be notified when it's ready.

I had the same problem, so this post really helped me. Thanks a lot! Also as feedback: The course was really good. I really appreciate that it was approachable and free. I will definately do the second one at some point.
